app/views/admin/users/index/_member_image.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="<%= admin_user_path(user.id) %>" class="mr-2">
```

Comment:
It seems very granular to have a partial just for this image, but the invitations & GDPR requests tables layout the image/member data slightly differently